src/com/google/javascript/jscomp/gwt/client/GwtRunner.java

```diff
@@ -461,8 +520,7 @@ private static void disableUnsupportedOptions(CompilerOptions options) {
         if (path == null) {
           path = unknownPrefix + i;
         }
-        path += ".map";
-        SourceFile sf = SourceFile.fromCode(path, file.sourceMap);
+        SourceFile sf = SourceFile.fromCode(path + ".map", file.sourceMap);
         inputSourceMaps.put(path, new SourceMapInput(sf));
```

Comment:
nice catch. I guess we were specifying that the source map was for the `file.js.map` file :|

Reply:
That was only one of the problems with source maps ...